src/update/verify_test.go

```diff
@@ -2,67 +2,78 @@ package update
 
 import (
 	"bytes"
+	"crypto"
 	"io"
 	"os"
+	"path/filepath"
 	"testing"
 
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
+	"github.com/sigstore/sigstore/pkg/signature"
 	"github.com/stretchr/testify/assert"
 )
 
+var goodKey = loadKey("good.pem")
+var badKey = loadKey("bad.pem")
+
+var goodKeyPub = loadKey("good.pub.pem")
+
+func loadKey(name string) []byte {
+	key, err := os.ReadFile(filepath.Join("src/update/test_data", name))
+	if err != nil {
+		panic(err)
+	}
+	return key
+}
+
+func sign(key []byte, msg []byte) io.Reader {
+	priv, err := cryptoutils.UnmarshalPEMToPrivateKey(key, cryptoutils.SkipPassword)
+	if err != nil {
+		panic(err)
+	}
+
+	verifier, err := signature.LoadSignerVerifier(priv, crypto.SHA256)
+	if err != nil {
+		panic(err)
+	}
+
+	bs, err := verifier.SignMessage(bytes.NewReader(msg))
+	if err != nil {
+		panic(err)
+	}
+
+	return bytes.NewReader(bs)
+}
+
 // readFile reads a file into an io.Reader
```

Comment:
```suggestion
// readFile reads a file into a byte slice.
```

This needs updating now that the type signature has changed.